### ISSUE TITLE
Make sure application can be run against Notes.app databases that don't have all columns in their schema

### ISSUE THREAD

**Issue by alexkolson** [HamburgChimps/apple-notes-liberator#47]
I broke this/introduced this bug trying to fix #40.

**Comment by urosch** [comment]
Cannot invoke "java.lang.Integer.intValue()" because "rowIndex" is null

is this the same error?

**Comment by aikinai** [comment]
I also get `Cannot invoke "java.lang.Integer.intValue()" because "rowIndex" is null`. Any chance to fix this?

**Comment by alexkolson** [comment]
Yeah I'm trying to get a note that will reproduce this. Have not given up yet!

**Comment by aikinai** [comment]
If it's the same issue as my #51, is there any way I can get logs to see where it's failing and then share the bad note with you?

**Comment by alexkolson** [comment]
It is indeed the same issue. To enable logging, [follow the instructions in the README](https://github.com/HamburgChimps/apple-notes-liberator#enable-logging). I always appreciate any additional information you can offer! Right now the logs do not indicate what note they fail on, so it might be a bit hard to pin point what exact note is causing the issue in your case. It would be a note that contains a table though for sure, as #51 only happens when processing tables in notes.
